websites/K/KaniNetwork/presence.ts

```diff
@@ -4,40 +4,42 @@ const presence = new Presence({
 
 presence.on("UpdateData", async () => {
   const presenceData: PresenceData = {
-      largeImageKey: "logo"
+      largeImageKey: "logo",
+      smallImageKey: "logo",
+      smallImageText: "KaniShiel"
     },
     browsingStamp = Math.floor(Date.now() / 1000),
     privacy = await presence.getSetting("privacy"),
     sprivacy = await presence.getSetting("super-privacy");
   presenceData.startTimestamp = browsingStamp;
-  if (sprivacy) {
+  if (sprivacy || window.location.pathname === ':8080') {
```

Comment:
:8080 ?

Reply:
We're using the same domain for a PufferPanel